src/solutions/ode_solutions.jl

```diff
@@ -15,7 +15,40 @@ struct ODESolution{T,N,uType,uType2,DType,tType,rateType,P,A,IType,DE} <: Abstra
   destats::DE
   retcode::Symbol
 end
-(sol::ODESolution)(t,deriv::Type=Val{0};idxs=nothing,continuity=:left) = sol.interp(t,idxs,deriv,sol.prob.p,continuity)
+
+function (sol::ODESolution)(t,deriv::Type=Val{0};idxs=nothing,continuity=:left)
+  if deriv != Val{0}
+    error("Not Implemented")
+  end
+
+  if idxs === nothing
```

Comment:
It seems it would be easier to extend if we would dispatch on `idxs` and `t`.

Reply:
Yes. That seems more practical. Thanks! Will change it.